index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+        <title>Setup & Mobile Portfolio Skeleton</title>
+</head>
+<body>
+    <main>
+        <header>
+            <nav class="navbar">
+            <h1>My Logo</h1>
```

Comment:
- [ ] Kindly make the logo an anchor tag `a` so that it can act as a hyperlink switching back to the home page